pandas/tests/groupby/test_groupby.py

```diff
@@ -1698,3 +1698,22 @@ def test_groupby_agg_ohlc_non_first():
     result = df.groupby(pd.Grouper(freq='D')).agg(['sum', 'ohlc'])
 
     tm.assert_frame_equal(result, expected)
+
+
+def test_groupby_multiindex_nat():
+    # GH 9236
+    midx = pd.MultiIndex(
```

Comment:
We typically don't construct MultiIndices in this fashion - can you alternately just use `from_tuples`?

Reply:
Sure. Will make the change tonight.